test/staking/Integration.ts

```diff
@@ -687,6 +707,15 @@ describe("Integration", function () {
     let mpondDelegatedByMaticDelegators: string[] = [];
 
     saveAndRestoreStateToParent(async () => {
+      // skip epoch till switching
+      let currentEpoch = (await clusterSelectors[0].getCurrentEpoch()).toNumber();
+      const switchingEpoch = BN.from(33).mul(86400).div(4 * 3600).toNumber() // coded as private variables in contract
```

Comment:
what does this switching thing do ?

Reply:
moves ahead in time if there switching period is enabled.